src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -516,6 +516,17 @@ public function iCheckItemsOfMyCart(string $tokenValue): void
         $this->cartsClient->executeCustomRequest($request);
     }
 
+    /**
+     * @Then /^my cart should have ("[^"]+") items total$/
```

Comment:
`/^my cart should have ("[^"]+") items total$/` -> `my cart should have :itemsTotal items total`?

Reply:
Argument 1 passed to Sylius\Behat\Context\Api\Shop\CartContext::myCartShouldHaveItemsTotal() must be of the type int, string given 🎉

Reply:
I don't think this will work at first it will return (using 12.34 as example) $12.34 but i need 1234 as API returns int number.